Content.Client/Power/Substation/SubstationVisualsSystem.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.GameObjects;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Content.Shared.Power.Substation;
+
+namespace Content.Client.Power.Substation;
+
+public sealed class SubstationVisualsSystem : VisualizerSystem<SubstationVisualsComponent>
+{
+    protected override void OnAppearanceChange(EntityUid uid, SubstationVisualsComponent component, ref AppearanceChangeEvent args)
+    {
+        if (args.Sprite == null || !args.Sprite.LayerMapTryGet(component.LayerMap, out var layer))
+            return;
+
+		if(args.AppearanceData.TryGetValue(SubstationVisuals.Screen, out var stateObject)
```

Comment:
```suggestion
        if (args.Sprite == null || !args.Sprite.LayerMapTryGet(component.LayerMap, out var layer))
            return;

		if (args.AppearanceData.TryGetValue(SubstationVisuals.Screen, out var stateObject)
```
Second if should have space before the parentheses, like the first if. Fix this in all the files, not just this one.

Reply:
There are tabs here also on the second if. Please make sure your code only uses 4 space indentations.